tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -2357,6 +2357,28 @@ def test_forward_one_hot():
     _test_forward_one_hot((3, 2, 4, 5), 6, 1, 0, 1, "int32")
     _test_forward_one_hot((3, 2, 4, 5), 6, 1.0, 0.0, 0, "float32")
 
+#######################################################################
+# AddN
+# ----------------------
+def _test_forward_add_n(inputs):
+    tf.reset_default_graph()
+    with tf.Graph().as_default():
+        temp = []
+        for each in inputs:
+            temp.append(tf.placeholder(shape=each.shape, dtype=each.dtype))
+        output = tf.add_n(temp)
+        compare_tf_with_tvm([each for each in inputs],[each.name for each in inputs], output.name)
```

Comment:
CI failed due to numpy ndarray has no name attribute. I guess you meant `each.name for each in temp` here?

Reply:
yep, thx for ur suggestions